README.md

```diff
@@ -108,6 +108,8 @@ Prior to your first deployment, you'll need to do a few things:
   fly volumes create data --size 1 --app indie-stack-template-staging
   ```
 
+- Update the `FLY_PRIMARY_REGION` environment variable in `fly.toml`. Set it to the same region as your volume (`fly volumes list`)
```

Comment:
This is another step that would be easy for people to skip. Maybe we should figure out how to add this question to the remix.init/index.js script in this repo so people are less likely to get it wrong...

Reply:
Oh that sounds nice. When does the init script run?